business_objects/record.py

```diff
@@ -194,6 +207,10 @@ def update_bytes_of_record_tokenized(
 
 
 def update_columns_of_tokenized_records(rt_ids: str, attribute_name: str) -> None:
+    # rt_ids = prevent_sql_injection(rt_ids, isinstance(rt_ids, str)) # excluded since prepared with ()
```

Comment:
How does the preperation with "()" avoid something like "(" + "'1' OR '1'='1') AND ('2'='2'" + ")"

Reply:
maybe misleading comment. this isn't relevant as the rt_ids string is prepared beforehand. (in refinery-tokenizer)
![image](https://github.com/code-kern-ai/refinery-submodule-model/assets/91723236/95caaf27-62f8-4840-979d-bffb68279c62)

the base assumtion on which the prevent method works is that it's a value not a full sql string or similar. So maybe as comment
`excluded since already prepared beforehand`